libraries/AP_BattMonitor/AP_BattMonitor.h

```diff
@@ -80,6 +82,7 @@ class AP_BattMonitor
         NeoDesign                  = 15,
         MAXELL                     = 16,
         Generator                  = 17,
+        Rotoye                     = 18,
```

Comment:
Can I be really cheeky and request that this gets bumped on to 19.  I have another PR which splits Generator into two backends so it would be nice to keep the two generator backends together.  PR#15626